File: Assignment 5/PythonAssignment5/Median.py
from math import ceil


class Median:

    def get_median(self, input_array, size):
        if input_array is None or size is None:
            return None
        else:
            input_array.sort()
            temp = size/2
            position = int(ceil(temp))
            return input_array[position - 1]

File: Assignment5/TypscriptAssignment5/src/Median.ts
export class Median{

    medianFinder(arrayInput: number[]): number {   
        var size =arrayInput.length 
        var sortedArray: number[] = arrayInput.sort((n1,n2) => n1 - n2);
        var position = Math.ceil(size/2)
        return sortedArray[position-1];
    }
}